riocli/device/metric.py

```diff
@@ -32,8 +32,10 @@
     help_options_color='green',
 )
 def device_metrics():
-    """
-    Device Metrics
+    """Interact with device metrics.
```

Comment:
todo: These APIs are unsupported now. We can remove these commands.

Reply:
Done.